test/screenshotter/ss_data.yaml

```diff
@@ -57,6 +57,7 @@ BinCancellation: |
     \end{array}
 BinomTest: \dbinom{a}{b}\tbinom{a}{b}^{\binom{a}{b}+17}
 BoldSpacing: \mathbf{A}^2+\mathbf{B}_3*\mathscr{C}'
+BoldSymbol: \boldsymbol{\omega}+\boldsymbol{\Omega}+\boldsymbol{A}+\boldsymbol{x}
```

Comment:
You'll need to generate screenshots and include those files as well.  Let me know if you have issues running `make screenshots`.

Reply:
`make screenshots` does not work. I get the following:

```
/Users/aterenin/Git/KaTeX/node_modules/selenium-webdriver/lib/promise.js:654
    throw error;
    ^

Error: Timed out waiting for the WebDriver server at http://127.0.0.1:49315/hub
    at onError (/Users/aterenin/Git/KaTeX/node_modules/selenium-webdriver/http/util.js
:87:11)
...
```

Reply:
@aterenin did `docker` start correctly?

Reply:
I am not familiar with docker, but I think so. How can I verify that?

Reply:
Here is the full terminal output:

```
MacBook-Pro:KaTeX aterenin$ make screenshots
dockers/Screenshotter/screenshotter.sh
~/Git/KaTeX/test/screenshotter/unicode-fonts ~/Git/KaTeX
HEAD is now at 99fa66a... Add fonts used by KaTeX screenshot tests
~/Git/KaTeX
Starting container for selenium/standalone-firefox:2.48.2
Container Saved file t started, creating screenshots...
Selenium driver in local session
(node:6108) [DEP0022] DeprecationWarning: os.tmpDir() is deprecated. Use os.tmpdir() i
nstead.
/Users/aterenin/Git/KaTeX/node_modules/selenium-webdriver/lib/promise.js:654
    throw error;
    ^

Error: Timed out waiting for the WebDriver server at http://127.0.0.1:60770/hub

    at onError (/Users/aterenin/Git/KaTeX/node_modules/selenium-webdriver/http/util.js
:87:11)
    at ManagedPromise.invokeCallback_ (/Users/aterenin/Git/KaTeX/node_modules/selenium
-webdriver/lib/promise.js:1379:14)
    at TaskQueue.execute_ (/Users/aterenin/Git/KaTeX/node_modules/selenium-webdriver/l
ib/promise.js:2913:14)
    at TaskQueue.executeNext_ (/Users/aterenin/Git/KaTeX/node_modules/selenium-webdriv
er/lib/promise.js:2896:21)
    at asyncRun (/Users/aterenin/Git/KaTeX/node_modules/selenium-webdriver/lib/promise
.js:2775:27)
    at /Users/aterenin/Git/KaTeX/node_modules/selenium-webdriver/lib/promise.js:639:7
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
From: Task: WebDriver.createSession()
    at Function.createSession (/Users/aterenin/Git/KaTeX/node_modules/selenium-webdriv
er/lib/webdriver.js:329:24)
    at new Driver (/Users/aterenin/Git/KaTeX/node_modules/selenium-webdriver/firefox/i
ndex.js:386:38)
    at Builder.build (/Users/aterenin/Git/KaTeX/node_modules/selenium-webdriver/builde
r.js:467:16)
    at buildDriver (/Users/aterenin/Git/KaTeX/dockers/Screenshotter/screenshotter.js:2
55:22)
    at _combinedTickCallback (internal/process/next_tick.js:131:7)
    at process._tickCallback (internal/process/next_tick.js:180:9)
    at Function.Module.runMain (module.js:678:11)
    at startup (bootstrap_node.js:187:16)
    at bootstrap_node.js:608:3
From: Task: WebDriver.manage().timeouts().setScriptTimeout(3000)
    at Driver.schedule (/Users/aterenin/Git/KaTeX/node_modules/selenium-webdriver/lib/
webdriver.js:377:17)
    at Timeouts.setScriptTimeout (/Users/aterenin/Git/KaTeX/node_modules/selenium-webd

river/lib/webdriver.js:1303:25)
    at buildDriver (/Users/aterenin/Git/KaTeX/dockers/Screenshotter/screenshotter.js:2
56:32)
    at _combinedTickCallback (internal/process/next_tick.js:131:7)
    at process._tickCallback (internal/process/next_tick.js:180:9)
    at Function.Module.runMain (module.js:678:11)
    at startup (bootstrap_node.js:187:16)
    at bootstrap_node.js:608:3
Failed taking screenshots, stopping and removing Saved file t
Starting container for selenium/standalone-chrome:2.48.2
Container Saved file t started, creating screenshots...
Selenium driver in local session
(node:6122) [DEP0022] DeprecationWarning: os.tmpDir() is deprecated. Use os.tmpdir() i
nstead.
/Users/aterenin/Git/KaTeX/node_modules/selenium-webdriver/chrome.js:185
      throw Error(
      ^

Error: The ChromeDriver could not be found on the current PATH. Please download the la
test version of the ChromeDriver from http://chromedriver.storage.googleapis.com/index
.html and ensure it can be found on your PATH.

    at new ServiceBuilder (/Users/aterenin/Git/KaTeX/node_modules/selenium-webdriver/c
hrome.js:185:13)
    at getDefaultService (/Users/aterenin/Git/KaTeX/node_modules/selenium-webdriver/ch
rome.js:362:22)
    at new Driver (/Users/aterenin/Git/KaTeX/node_modules/selenium-webdriver/chrome.js
:771:34)
    at Builder.build (/Users/aterenin/Git/KaTeX/node_modules/selenium-webdriver/builde
r.js:464:16)
    at buildDriver (/Users/aterenin/Git/KaTeX/dockers/Screenshotter/screenshotter.js:2
55:22)
    at _combinedTickCallback (internal/process/next_tick.js:131:7)
    at process._tickCallback (internal/process/next_tick.js:180:9)
    at Function.Module.runMain (module.js:678:11)
    at startup (bootstrap_node.js:187:16)
    at bootstrap_node.js:608:3
Failed taking screenshots, stopping and removing Saved file t
make: *** [screenshots] Error 1
```

Reply:
To add one final note: `docker` was installed via `npm install -g docker`

Reply:
Figured out the problem: I previously didn't install the correct `docker`.

Screenshots added and pushed.

Reply:
@aterenin cool.  I take it `npm install docker` installs a client.  I should update the README.md for the Screenshot to point to link docker's download page or homepage so there's less confusion.

Reply:
It might be good to include some other symbols in this such as `\sum` and `\int`.

Reply:
Done